cinch/roles/jenkins_slave/templates/swarm.upstart.conf

```diff
@@ -0,0 +1,21 @@
+description "Jenkins slave swarm client"
```

Comment:
Ansible convention seems to go with the .j2 extension but I avoid it. With the .j2 extension basically no text editor around will pick up proper syntax highlighting for the file. I've found that many of them will still get the highlighting correct if you just leave off the .j2 extension. (It's not foolproof with vim, since some files are picked up by their location on the filesystem, rather than extension, but it has a much greater chance of success than with the .j2 extension - therefore, I always eschew the extension)